sim/Makefile

```diff
@@ -13,16 +17,18 @@ TARGET_PROJECT ?= firesim
 
 # Users can override this to point at a makefrag defined in a parent project
 # that submodules firesim or source sim/Makefrag directly
-TARGET_PROJECT_MAKEFRAG ?= src/main/makefrag/$(TARGET_PROJECT)/Makefrag
+TARGET_PROJECT_MAKEFRAG ?= $(firesim_base_dir)/src/main/makefrag/$(TARGET_PROJECT)/Makefrag
 
-default: compile
+ifdef FIRESIM_STANDALONE
+
+$(info Firesim standalone)
+
+base_dir := $(firesim_base_dir)
 
 SBT ?= sbt
```

Comment:
Is it intentional that `SBT` and related vars are not defined in the `else`?

Reply:
Yeah, they are defined in rebar's makefrag.